src/plugins/usage_collection/server/routes/stats.ts

```diff
@@ -19,6 +19,7 @@
 
 import { schema } from '@kbn/config-schema';
 import { i18n } from '@kbn/i18n';
+
```

Comment:
This is the only change related to the Kibana Telemetry team 😅

Reply:
Should the comment go in here as well?

Reply:
This is in server code so there shouldn't be anything changed here; this was probably related to something that's since been reverted. @brittanyjoiner15 can you revert the change in this file please?